airflow/providers/databricks/PROVIDERS_CHANGES_2020.06.24.md

```diff
@@ -9,7 +9,7 @@
 | [f6bd817a3](https://github.com/apache/airflow/commit/f6bd817a3aac0a16430fc2e3d59c1f17a69a15ac) | 2020-06-16  | Introduce &#39;transfers&#39; packages (#9320)                                                                                                                             |
```

Comment:
Also all those PROVIDERS_CHANGES are referring to the commits so I would not change those. There are many provider's changes files below - all of them should be exempted from that change.